### PR TITLE
Aizad/Fix Derived toggle click on reload

### PR DESCRIPTION
**Pull request by aizad-deriv**

### CODE REVIEW

File: src/javascript/app/pages/trade/markets.jsx [hunk 1]
```diff
@@ -536,11 +538,12 @@ class Markets extends React.Component {
...
                                                     className={classNames('market', {
-                                                        'active': subgroup_active,
+                                                        'active'         : subgroup_active,
+                                                        'accordion-label': !!open_accordion,
                                                     })}
-                                                    onClick={toggleAccordion || (subgroup_active ? toggleAccordion : '')}
+                                                    onClick={toggleAccordion}
                                                 >
-                                                    <span className={`icon synthetic_index ${open_accordion ? 'active' : ''}`} />
+                                                    <span className={`icon synthetic_index ${subgroup_active ? 'active' : ''}`} />
```
Comment (ameerul-deriv) [inline src/javascript/app/pages/trade/markets.jsx:546]: cant we write this similar to line 541 using classNames function

File: src/javascript/app/pages/trade/markets.jsx [hunk 2]
```diff
@@ -536,11 +538,12 @@ class Markets extends React.Component {
...
                                                     <span>{group_markets[item].markets[0].subgroup_name}</span>
                                                     <span className={`accordion-icon icon ${open_accordion ? 'active' : ''}`} />
```
Comment (ameerul-deriv) [inline src/javascript/app/pages/trade/markets.jsx:548]: same here

File: src/javascript/app/pages/trade/markets.jsx (hunk 1 above)
Comment (aizad-deriv) [reply src/javascript/app/pages/trade/markets.jsx]: Resooolved 😩😩😩

File: src/javascript/app/pages/trade/markets.jsx (hunk 2 above)
Comment (aizad-deriv) [reply src/javascript/app/pages/trade/markets.jsx]: Resooolved 😩😩😩

**Review by ameerul-deriv — APPROVED** [review]
LGTM

**Review by akmal-binary — APPROVED** [review]

**Review by farabi-deriv — APPROVED** [review]

**Pull request merged**